src/crimson/os/seastore/segment_manager/zbd.cc

```diff
@@ -200,6 +212,31 @@ static seastar::future<size_t> get_zone_capacity(
   );
 }
 
+static seastar::future<size_t> get_nr_cnv_zones(
```

Comment:
Add some comments:
```
// get the number of conventional zones in case of SMR HDD,
// they are randomly writable and don't respond to zone operations
```

Reply:
Added.